Cargo.toml

```diff
@@ -19,6 +19,7 @@ categories = [ "no-std", "embedded", "wasm", "parser-implementations" ]
 [dependencies]
 num-traits = { version = "0.2.11", default-features = false }
 smallvec = { version = "1.4.1", default-features = false }
+rust_decimal = { version = "1.7", default-features = false}
```

Comment:
My understanding is that `optional` is for platform specific dependencies no? See the [Cargo docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#platform-specific-dependencies)

Reply:
Not just for platform dependencies.  If a crate is only used by a feature, then it can be marked optional so it doesn't get pulled in during a normal build.

Reply:
Also, there is a serde feature for rust_decimal. Rhai's serde feature should include rust_decimal/serde in its features set.

Reply:
And currently it doesn't build the `no_std` sample.  However, it could be due to some simple code duplication...

Reply:
Ah I see how to use optional dependencies now. :wink: